packages/riipen-ui/src/components/Form.jsx

```diff
@@ -9,80 +9,37 @@ import withClasses from "../utils/withClasses";
 import Notice from "./Notice";
 import Typography from "./Typography";
 
-class Form extends React.Component {
-  static displayName = "Form";
-
-  static propTypes = {
-    /**
-     * The content of the component.
-     */
-    children: PropTypes.node,
-
-    /**
-     * An array of custom CSS classes to apply.
-     */
-    classes: PropTypes.array,
-
-    /**
-     * Top level form error to display. One of either string or react node.
-     */
-    error: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
-
-    /**
-     * A list of form errors to display. One of an object consisting of key value errors,
-     * an array of strings, or an array of react nodes.
-     */
-    errors: PropTypes.oneOfType([
-      PropTypes.arrayOf(PropTypes.string),
-      PropTypes.arrayOf(PropTypes.element),
-      PropTypes.object
-    ]),
-
-    /**
-     * If `false`, error will not be scrolled to when it is passed to the form.
-     */
-    errorScroll: PropTypes.bool,
-
-    /**
-     * If `false`, pressing enter to submit the form will be disabled.
-     */
-    enter: PropTypes.bool
-  };
-
-  static defaultProps = {
-    classes: [],
-    enter: true,
-    errorScroll: true
-  };
-
-  componentDidUpdate(prevProps) {
-    const theme = this.context;
-
-    // If an error id dynamically added to the form, scroll the error notice in to view
-    if (
-      !prevProps.error &&
-      !prevProps.errors &&
-      (this.props.error || this.props.errors) &&
-      this.error &&
-      this.props.errorScroll
```

Comment:
There was a bug here as props.errors is an object and therefor always true once it had been set. So !prevProps.errors was always false